nltk/translate/util.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+
+def jacknifing(score_list, averaging=True):
+        ''' This is a averaging function for calculating ROUGE score
+    when multiple references are present. This technique has been
+    referred to as the Jacknifing technique in the original paper.
+    score_list = list of scores over which averaging occurs
+    '''
+        if(len(score_list) == 1):
+            return np.mean(score_list)
+        elif((len(score_list) > 1) and (averaging is False)):
+            return score_list
+        else:
+            average = []
+            for i in score_list:
+                dummy = [j for j in score_list if i != j]
```

Comment:
Please add some comments so to explain what's `dummy` in this scenario.

Reply:
Done